exercises/secret-handshake/secret-handshake.spec.js

```diff
@@ -38,7 +38,8 @@ describe('Secret Handshake', function () {
 
   xit('text is an invalid secret handshake', function () {
     expect( function () {
-      var handshake = new SecretHandshake('piggies');
+      /* eslint no-unused-vars: ["error", { "varsIgnorePattern": "[iI]gnored" }]*/
+      var ignoredHandshake = new SecretHandshake('piggies');
```

Comment:
So, the idea here is to cause an exception to be thrown.  What do you think happens to `ignoredHandshake` if `new SecretHandshake()` throws an exception?  Do we care?  Is there a way you could simplify lines 41 and 42?

Reply:
For this, I was getting the no-unused-vars error from the linter. We need to create the variable to run the test, but we don't actually use the variable in the test. I'm not sure if there's a simpler way to use the variable and/or exclude it: https://eslint.org/docs/rules/no-unused-vars#varsignorepattern. This is my first time working with this linter, so I might be missing something. What do you think?

Reply:
I think all you need to invoke the constructor by calling:

```javascript
SecretHandshake('piggies')
```

No need to assign it to anything, or annotate the method for the linter.

Reply:
Running into a new error with this method: https://eslint.org/docs/rules/no-new. Looking into potential solutions outside varsIgnorePattern.

Reply:
@joefitz12 oops...that's my mistake.  I guess that leaves us with the annotation...